frontend/src/scenes/sessions/SessionDetails.tsx

```diff
@@ -90,6 +93,23 @@ export function SessionDetails({ session }: { session: SessionType }): JSX.Eleme
                 },
                 rowExpandable: (event) => !!event,
                 expandRowByClick: true,
+                columnWidth: ANTD_EXPAND_BUTTON_WIDTH + MATCHING_EVENT_ICON_SIZE,
+                expandIcon: function _renderExpandIcon(expandProps) {
+                    const { record: event } = expandProps
+                    return (
+                        <ExpandIcon {...expandProps}>
+                            {(session.matching_events || []).includes(event.id) ? (
```

Comment:
Note: You're introducing n^2 search here which can cause issues whenever someone is using a slower computer on a large project. Is it guaranteed that both arrays are small?

If not, worth considering using `Set` object.

Reply:
makes sense! I think it's worth changing because we perform the same computation in line 76